test/configs/certs/tlsauth/delete-cert-from-store.ps1

```diff
@@ -0,0 +1,2 @@
+$issuer="Synadia Communications Inc."

```

Comment:
Maybe put all these in their own sub-dir.

Reply:
Yes, don't want to jump-scare the poor Gophers with PowerShell. I will hide away in a sub-dir.